kombu/transport/kafka.py

```diff
@@ -0,0 +1,336 @@
+"""
+kombu.transport.kafka
```

Comment:
I totally agree with naming it ``pykafka``. Would be nice to have a ``kafka_python`` driver in the future.